recipes-tag/bluesky-kafka/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "0.1" %}
+
+package:
+    name: bluesky-kafka
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/nsls-ii/bluesky-kafka
```

Comment:
Should we also exploit the `{{ name }}` feature as in other recent updates?
Maybe use `NSLS-II` instead of `nsls-ii`?

Reply:
thanks! updated